docs/streaming/usage-guide.mdx

```diff
@@ -0,0 +1,607 @@
+---
+title: "Usage Guide"
+description: "A deep dive into streaming agents"
+---
+
+Let's build a simple SQL generation agent network with realtime streaming. To kick things off, let's walk through a few endpoints you'll need to wire this all up:
+
+- **Inngest Client (`/api/inngest/client.ts`)**: Initializes Inngest with the `realtimeMiddleware`.
+- **Realtime Channel (`/api/inngest/realtime.ts`)**: Defines a typed realtime channel and topic.
+- **Chat Route: `/api/chat/route.ts`**: This is a standard Next.js API route. Its only job is to receive a request from the frontend and send an event to Inngest to trigger a function.
+- **Token Route: `/api/realtime/token/route.ts`**: This secure endpoint generates a subscription token that the frontend needs to connect to Inngest realtime.
+- **Inngest Route: `/api/inngest/route.ts`**: The standard handler that serves all your Inngest functions.
+
+Let's take a closer look at each of these endpoints and what they do...
```

Comment:
We should add an `npm install` here at the top to show that the `@inngest/realtime` package is also required